components/coda/actions/copy-doc/copy-doc.mjs

```diff
@@ -0,0 +1,41 @@
+import coda from "../../coda.app.mjs";
+
+export default {
+  key: "coda_copy-doc",
+  name: "Copy Doc",
+  description: "Creates a copy of the specified Coda doc",
+  version: "0.0.17",
```

Comment:
This is hard when you're developing, but before you merge, you'll want to reset these to `0.0.1`: https://pipedream.com/docs/components/guidelines/#versioning .

We're planning to introduce a `pd dev` command for actions so that you don't have to constantly version to test.

Reply:
Issue addressed

Reply:
Yes that generally should work fine